tools/syz-covermerger/init_db.sh

```diff
@@ -19,6 +19,7 @@ CREATE TABLE
     "covered" bigint,
     "linesinstrumented" bigint[],
     "hitcounts" bigint[],
+    "manager" text,
```

Comment:
Did you consider making `manager` a part of `merge_history` instead? What was your reasoning?

Reply:
In a negative way. The "session" is the aggregation session and the was DB designed in a way to have only 1 `merge_history` record per session.
It'll be a quite fundamental change.

`files` is a leaf table and is way more flexible.